.github/ISSUE_TEMPLATE/2_feature_request.yml

```diff
@@ -15,9 +15,9 @@ body:
     attributes:
       label: Screenshot
       description: You can provide screenshots/mockups to better visualize your idea. Files can be dropped in this field
-  - type: input
+  - type: textarea
     attributes:
-      label: Example URL
+      label: Example URLs
```

Comment:
This should also already be covered by

- https://github.com/refined-github/refined-github/pull/5498